nyt-top-20-science-news/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
```

Comment:
I'm sure this is boilerplate, but I was interested to know what these files were, since I hadn't heard of `.pnp` or `.pnp.js` before (or why someone would want to git ignore js files). My searching took me to [Patterns and Practices client side libraries](https://pnp.github.io/pnpjs/), which seems to be used to interact with online MS products, like SharePoint & Office 365. If that is what these entries refer to I'm extra confused why that would be included with default React boilerplate. Is this a holdover from a previous project you worked on? If so, was it easy to meaningfully interact with the MS products in your project?

Reply:
This is a boilerplate from running `create-react-app`. Interesting read, thanks! Indeed odd that it's included. I didn't use it for this project but sometimes I use this app to [generate a .gitignore](https://www.toptal.com/developers/gitignore).